selftest/cgroup-legacy/main.go

```diff
@@ -40,13 +40,14 @@ func main() {
 
 	cgroupRootDir := getCgroupV2RootDir()
 
+	// link, err := prog.AttachCgroup(cgroupRootDir)
```

Comment:
Intentional?

Reply:
yep, wanted to show it as an alternative!